packages/common/src/ArcGISContext.ts

```diff
@@ -21,6 +21,7 @@ const hubApiEndpoints = {
   domains: "/api/v3/domains",
   search: "/api/v3/datasets",
   discussions: "/api/discussions/v1",
+  ogcapi: "/api/search/v1",
```

Comment:
What about `ogcRecords`? Just in case we decide to follow the OGC features spec down the line.